pkg/agent/agent.go

```diff
@@ -272,9 +297,81 @@ func (s *NimbessAgent) Add(ctx context.Context, req *cni.CNIRequest) (*cni.CNIRe
 		log.Infof("Kernel Virtual Port created: %v in meta network: %s", res.GetInterfaces(), metaKey)
 	}
 
+	// Create FastPath secondary Network
```

Comment:
remove "secondary"

Reply:
will remove and make a TODO note to remove creating the network/pipeline automatically. Only the Port will be created in the future unless a UNP applies to this pod.